Cython/Compiler/MemoryView.py

```diff
@@ -841,17 +844,35 @@ def use_cython_array_utility_code(env):
     requires=[],  # require cython_array_utility_code
 )
 
-view_utility_code = load_memview_cy_utility(
+memoryview_utility_code = load_memview_cy_utility(
         "View.MemoryView",
         context=context,
-        requires=[Buffer.buffer_struct_declare_code,
+        requires=[
+                  Buffer.buffer_struct_declare_code,
                   Buffer.buffer_formats_declare_code,
                   memviewslice_init_code,
                   is_contig_utility,
                   overlapping_utility,
                   copy_contents_new_utility,
                   ],
 )
+
+if Options.shared_utility_qualified_name:
```

Comment:
Something like this won't work well. Users cannot be required to set the option attribute before importing Cython.